### ISSUE TITLE
jinja2 inventory_path missing

### ISSUE THREAD

**Issue by sebradloff** [kapicorp/kapitan#776]
**Describe the bug/feature**
I believe jinja2 templating might have issues with inventory similar to this issue with Kadet a few months ago: https://github.com/kapicorp/kapitan/pull/664

**To Reproduce**
I'm not sure how to reproduce at this time

**Expected behavior**
Inventory path to not be an issue. Last worked for me on `v0.29.2`

**Screenshots**
```
root@8ed7e8a1e9cf:/src# kapitan c -t box-0
Unknown (Non-Kapitan) Error occurred
multiprocessing.pool.RemoteTraceback:
"""
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "/opt/venv/lib/python3.7/site-packages/kapitan/targets.py", line 464, in compile_target
    input_compiler.compile_obj(comp_obj, ext_vars, **kwargs)
  File "/opt/venv/lib/python3.7/site-packages/kapitan/inputs/base.py", line 55, in compile_obj
    self.compile_input_path(input_path, comp_obj, ext_vars, **kwargs)
  File "/opt/venv/lib/python3.7/site-packages/kapitan/inputs/base.py", line 77, in compile_input_path
    **kwargs,
  File "/opt/venv/lib/python3.7/site-packages/kapitan/inputs/jinja2.py", line 39, in compile_file
    context["inventory"] = inventory(self.search_paths, target_name)
  File "/opt/venv/lib/python3.7/site-packages/kapitan/resources.py", line 264, in inventory
    inventory_path = inventory_path_arg.inventory_path
AttributeError: 'NoneType' object has no attribute 'inventory_path'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/venv/lib/python3.7/site-packages/kapitan/targets.py", line 136, in compile_targets
    [p.get() for p in pool.imap_unordered(worker, target_objs) if p]
  File "/opt/venv/lib/python3.7/site-packages/kapitan/targets.py", line 136, in <listcomp>
    [p.get() for p in pool.imap_unordered(worker, target_objs) if p]
  File "/usr/local/lib/python3.7/multiprocessing/pool.py", line 748, in next
    raise value
AttributeError: 'NoneType' object has no attribute 'inventory_path'


'NoneType' object has no attribute 'inventory_path'
```

** If it's a bug (please complete the following information):**
docker image `v0.30.0-rc.0`

**Additional context**

**Comment by sebradloff** [comment]
Potential related issue: https://github.com/kapicorp/kapitan/issues/737

**Comment by sebradloff** [comment]
I think I figured out what's going on. I was able to resolve this issue by using the `--force` flag on the compile command. I was previously using kapitan v0.29.2 and I think the cache had different values and so it "errored" out. I believe this section of a PR was responsible for that change: https://github.com/kapicorp/kapitan/commit/6d23e000c03749671ed2d635c9102c289da5a8d6#diff-4e264181ddb7002a2fa328e13e91bbe1704658469683a47af40619390d411407L255-L256
